packages/ag-charts-community/src/chart/series/cartesian/histogramSeries.ts

```diff
@@ -691,14 +691,17 @@ export class HistogramSeries extends CartesianSeries<SeriesNodeDataContext<Histo
             })
         );
 
-        const addedIds: { [key: string]: boolean } = {};
-        diff.added.forEach((d: string[]) => {
-            addedIds[d.join('_')] = true;
-        });
-        const removedIds: { [key: string]: boolean } = {};
-        diff.removed.forEach((d: string[]) => {
-            removedIds[d.join('_')] = true;
-        });
+        // Zip an array into an object of keys with a given value
+        const zipObject = (props: Array<any>, value = true) => {
+            const zipped: { [key: string]: boolean } = {};
+            for (let i = 0; i < props.length; i++) {
+                zipped[`${props[i]}`] = value;
+            }
+            return zipped;
+        };
```

Comment:
Where did we discuss would be a good place for this kind of util?

Reply:
Perhaps in it's own module in https://github.com/ag-grid/ag-charts/tree/25ba7de8971ec1d70fc635c2cee359b051e9fb87/packages/ag-charts-community/src/util ?